src/ui/public/agg_response/tabify/_response_writer.js

```diff
@@ -75,7 +75,10 @@ export function TabbedAggResponseWriterProvider(Private) {
     if (group) {
       table.aggConfig = agg;
       table.key = key;
-      table.title =  (table.fieldFormatter()(key)) + ': ' + agg.makeLabel();
+      table.title = (table.fieldFormatter()(key));
+      if (agg.makeLabel() !== 'filters') {
```

Comment:
This will problem really affects any aggregation that does not implement `makeLabel` function and relies on the default implementation (which just returns the aggregation name). How about instead of hard coding to check against `filters` you check that agg.makeLabel() !== agg.name? A comment about why this check is needed would also be helpful.